curriculum/challenges/english/02-javascript-algorithms-and-data-structures/functional-programming/refactor-global-variables-out-of-functions.english.md

```diff
@@ -45,30 +48,24 @@ tests:
 // the global variable
 var bookList = ["The Hound of the Baskervilles", "On The Electrodynamics of Moving Bodies", "Philosophiæ Naturalis Principia Mathematica", "Disquisitiones Arithmeticae"];
 
-/* This function should add a book to the list and return the list */
-// New parameters should come before bookName
-
-// Add your code below this line
+// Change code below this line
 function add (bookName) {
 
   bookList.push(bookName);
   return bookList;
   
-  // Add your code above this line
+  // Change code above this line
 }
 
-/* This function should remove a book from the list and return the list */
-// New parameters should come before the bookName one
-
-// Add your code below this line
+// Change code below this line
 function remove (bookName) {
   var book_index = bookList.indexOf(bookName);
   if (book_index >= 0) {
 
     bookList.splice(book_index, 1);
     return bookList;
 
-    // Add your code above this line
+    // Change code above this line
```

Comment:
Should this begin with "Only...", being consistent with the other amended commits earlier in the PR?
```suggestion
  // Only change code above this line
```

Reply:
No, because this particular challenge has multiple places where we expect the user to add/modify code.  There are only a few challenges like this, which is why I have a separate comment for them.

Also, if you look at the rest of the lines in the file, you may see comments that were not changed.  Trust me that they are taken care of on a separate PR.  Also, Oliver's PR will create a warning if a comment is found that is not part of the translation dictionary that I have created.  There are about 50 unique comments, but there originally was 380.